inst/extdata/database/tedfs/Tech/Methanol Synthesis.csv

```diff
@@ -14,15 +17,20 @@ Input|Electricity,Output|Methanol,,#,1.07,,MJ,1,kg,,Soler22,Table 3
 Input|Electricity,Output|Methanol,2019,#,25.4,,GJ,1,metric_ton,,IEA-FOH19,Assumptions annex
 Input|Electricity,Output|Methanol,2030,#,23.7,,GJ,1,metric_ton,,IEA-FOH19,Assumptions annex
 Input|Electricity,Output|Methanol,,#,0.175,,kWh,1,kg,,Nyari20,Table 6
+Input|Electricity,Output|Methanol,,#,0.067,,kWh,1,kWh,Found in supplementary information,Stolz22,Table 5
```

Comment:
We seem to have quite a large variation w.r.t. the electricity demand here.

If I run `DataSet('Tech|Methanol Synthesis').normalise().query("variable=='Input|Electricity'")[['variable', 'value', 'unit', 'reference_variable', 'reference_value', 'reference_unit', 'source']]` on @clarabachorz/posted, I get the following output:
```
             variable     value unit reference_variable  reference_value reference_unit        source
13  Input|Electricity  0.268740  MWh    Output|Methanol              1.0        MWh;LHV  Bazzanella17
14  Input|Electricity  0.053250  MWh    Output|Methanol              1.0        MWh;LHV       Soler22
15  Input|Electricity  1.264072  MWh    Output|Methanol              1.0        MWh;LHV     IEA-FOH19
16  Input|Electricity  1.179468  MWh    Output|Methanol              1.0        MWh;LHV     IEA-FOH19
17  Input|Electricity  0.031353  MWh    Output|Methanol              1.0        MWh;LHV       Nyari20
18  Input|Electricity  0.067000  MWh    Output|Methanol              1.0        MWh;LHV       Stolz22
```

I supposed it depends on what you assume for the overall energy efficiency of the respective processes. For instance, the hydrogenation of CO₂ (i.e. CO₂ + H₂ → CH₃OH + H₂O) is, to the best of my knowledge, an exothermic reaction. If that waste heat is used for heating precursors, other energy demand (eg compression), or even producing electricity, then the overall electricity demand of the plant could be near zero or even negative.

It seems very hard to know these thing for certain. I'm just flagging this here. We may need to revisit this.

Reply:
Thanks for flagging this up! I'll add it to the list of things to look at.